Userland/Libraries/LibWeb/WebDriver/Capabilities.cpp

```diff
@@ -394,13 +394,13 @@ Response process_capabilities(JsonValue const& parameters)
 
 LadybirdOptions::LadybirdOptions(JsonObject const& capabilities)
```

Comment:
This reminds me, does Ladybird or libjs-test262 need any updates for this?

Reply:
Oops, I forgot to check that.

Reply:
OK, Ladybird did need fixing but it looks like libjs-test262 doesn't use any AK Json types.